src/main/java/de/tum/in/www1/artemis/web/rest/dto/RatingDTO.java

```diff
@@ -0,0 +1,10 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import de.tum.in.www1.artemis.domain.Rating;
+
+public record RatingDTO(Integer rating, DomainObjectIdDTO result) {
+
+    public RatingDTO(Rating rating) {
```

Comment:
Why is the rating an integer?

Reply:
`Rating` is class that contains an integer `rating`, a start-rating from 1 to 5.

Reply:
And is it null if no rating is given?

Reply:
I used `Integer` here simply to mirror the same type as used in the `Rating` class, I did not look into it in that detail. I'd expect that there is just no rating object in that case.